.github/workflows/cherryPick.yml

```diff
@@ -254,10 +254,11 @@ jobs:
           status: custom
           custom_payload: |
             {
-              "channel": "#deployer",
-              "attachments": [{
-                "color": "#DB4545",
-                "text": "🚨 Cherry-pick to ${{ inputs.TARGET }} has conflicts and requires manual resolution.\nOriginal PR: ${{ inputs.PULL_REQUEST_URL }}\nConflict PR: ${{ steps.createPullRequest.outputs.PR_URL }}"
+              channel: '#deployer',
+              attachments: [{
+                color: "#DB4545",
```

Comment:
NAB - I see that sometimes we use `color: 'danger'` instead, though it's not consistent.